### PR TITLE
Added the basic sample app for the FTCR routing

### PR DESCRIPTION
**Pull request by SiriUS2551AK**
Signed-off-by: Ihor Oliinyk <powerxm2551@gmail.com>

### CODE REVIEW

File: FTCR/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.here.android.example.ftcr">
+
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```
Comment (NazarKacharaba) [inline FTCR/app/src/main/AndroidManifest.xml:6]: Why we need this permission for FTCR app?

**Comment by SiriUS2551AK** [comment]
> Rename folder `FTCR`-> `ftcr`

Renamed.

File: FTCR/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+ */
+public class MapFragmentView {
+    private static final String TAG = MapFragmentView.class.getName();
+    private AndroidXMapFragment m_mapFragment;
+    private Button m_calculateRouteButton;
+    private AppCompatActivity m_activity;
+    private Map m_map;
+    private FTCRMapRoute m_mapRoute;
+
+    private GeoCoordinate[] fake_route;
```
[the remark this answers is not in the document]
Comment (SiriUS2551AK) [reply FTCR/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java]: Fixed.

File: FTCR/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+                     *
+                     */
+                    calculateRoute();
+                }
+            }
+        });
+
+    }
+
+    private void createRoute() {
```
[the remark this answers is not in the document]
Comment (SiriUS2551AK) [reply FTCR/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java]: Renamed.

File: ftcr/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+        /* Define waypoints for the route */
+        RouteWaypoint startPoint = new RouteWaypoint(m_startPoint);
+
+        RouteWaypoint destination = new RouteWaypoint(m_endPoint);
+
+        /* Initialize a RoutePlan */
+        List<RouteWaypoint> routePoints = new ArrayList<>();
+        routePoints.add(startPoint);
+        routePoints.add(destination);
+        FTCRRoutePlan routePlan = new FTCRRoutePlan(routePoints, DEFAULT_OVERLAY, routeOptions);
```
Comment (NazarKacharaba) [inline ftcr/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java:213]: Use following constructor FTCRRoutePlan(routePoints, routeOptions). Github examples use 3.16.3 SDK.

Comment (SiriUS2551AK) [reply ftcr/app/src/main/java/com/here/android/example/ftcr/MapFragmentView.java]: Fixed.

**Pull request merged**